components/Markets/ConditionalMarketCard.tsx

```diff
@@ -376,7 +378,7 @@ export function ConditionalMarketCard({ isPassMarket = false }: { isPassMarket?:
                   >
                     <Text size="xs">
                       Max{' '}
-                      {maxOrderAmount()
+                      {maxOrderAmount() && maxOrderAmount() < 1000
```

Comment:
I don't understand why restricting maxOrderAmount to be under 1000, is it for formating ?